### PR TITLE
Welcome video

### PR DESCRIPTION
**Pull request by fabriziovigevani**

### CODE REVIEW

File: src/components/Welcome/WelcomeLoader.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          padding: ${1 * GU}px;
+          background: ${theme.surface};
+          box-shadow: 0 1px 4px 0 rgb(0 0 0 / 6%), 0 2px 12px 0 rgb(0 0 0 / 16%);
+          border: 1px solid ${theme.border};
+          border-radius: 50%;
+          cursor: pointer;
+
+          &:hover {
+            background: ${theme.background};
+          }
```
Comment (0xGabi) [inline src/components/Welcome/WelcomeLoader.js:41]: Magic 🔮

File: src/components/Welcome/WelcomeModal.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                position: relative;
+                height: 0;
+              `}
+            >
+              <iframe
+                width="853"
+                height="480"
+                src={`https://www.youtube.com/embed/${EMBED_ID}`}
+                frameBorder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
```
Comment (0xGabi) [inline src/components/Welcome/WelcomeModal.js:32]: 😎

**Review by 0xGabi — APPROVED** [review]
Looking really good! 🙌

File: src/components/Welcome/WelcomeModal.js (hunk 2 above)
Comment (rperez89) [reply src/components/Welcome/WelcomeModal.js:32]: 🚗🚗🚗🚗🚗🚗

**Pull request merged**